packages/integrations/vercel/src/lib/nft.ts

```diff
@@ -1,6 +1,6 @@
-import type { AstroIntegrationLogger } from 'astro';
-import { relative, relative as relativePath } from 'node:path';
```

Comment:
Weird

Reply:
I submitted a bug in the Biome repository. The code, even though weird, is correct. Biome forgets the `as` import.

Hopefully this change won't break the adapter logic!